src/scenes/home/benefit/benefit.js

```diff
@@ -121,7 +122,7 @@ const Benefit = () => (
         </p>
       </div>
     </Section>
-    <Section title="Sponsors" theme="white">
+    <Section title="See Who's Joining Us!" theme="white">
```

Comment:
This is too long of a section header - on mobile the header will render as 2 lines in an ugly manner :(